console/console-init/ui/src/queries/Address.ts

```diff
@@ -0,0 +1,696 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import gql from "graphql-tag";
+import { ISortBy } from "@patternfly/react-table";
+
+const DELETE_ADDRESS = gql`
+  mutation delete_addr($a: ObjectMeta_v1_Input!) {
+    deleteAddress(input: $a)
+  }
+`;
+
+const PURGE_ADDRESS = gql`
+  mutation purge_addr($a: ObjectMeta_v1_Input!) {
+    purgeAddress(input: $a)
+  }
+`;
+
+const ALL_ADDRESS_FOR_ADDRESS_SPACE_FILTER = (
```

Comment:
I think we can modify this name as FILTER_FOR_ADDRESSES.

Reply:
done